### PR TITLE
Experiment Step docs

### PR DESCRIPTION
**Pull request by borisrev**
# Pull Request/Issue Resolution

**Description of Change:**
Adding documentation for Experiment Steps. Beta forthcoming. Docs to be hidden for now. Will submit separate PR to unhide and add to appropriate lists in parent pages when it goes GA.

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [x] Yes (Beta beginning 5/24, but please merge asap)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [x] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [x] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+nav_title: Experiment Step
+title: Audience Path Step
+alias: /audience_paths/
```
Comment (BrandonBrowning) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md:4]: fixme

**Review by BrandonBrowning — COMMENTED** [review]
will actually review soon, but thought I shoul

**Comment by borisrev** [comment]
How does :shipit: mean ship it?

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+# Experiment Step
+
+> Experiment Steps allow you to test multiple possible Canvas paths against each other and a control group at any point in the user journey.
+
+Experiment Step will randomly assign users to the different paths (or optional control group) you create. Portions of the audience will be assigned to the different paths according to percentages you select, allowing you to test different messages or entirely different paths against each other and then determine which to keep or eliminate. After launching, analytics will allow you to track performance and see whether it differs across the different paths to help you determine which path should get what proportion of users (or all of them!).
+
+![Experiment Step ][0]{: style="max-width:60%"}
+
+## Create an Experiment Step,
```
Comment (EVERROCKET) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md:18]: No comma in the end?

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+In the default Experiment Step shown to the right, there are already two default paths, __Path 1__ and __Path 2__, with 50% of the audience being sent down each path. Click on the Experiment Step itself to expand the Experiment Settings panel, and you''ll see the configuration options for the step. 
+
+### 1. Choose The Number of Paths and Audience Distribution
+
+![Experiment Settings][1]{: style="max-width:80%"}
+
+You can add up to 4 paths by clicking `Add Path` and an optional control group by checking the box labeled `Add a Control Group`. Using the input boxes above each path, you can input the percentage of the audience that should go to each path as well as the control group. Naturally, the percentages must add up to 100% to proceed. The 'Distribute Paths Evenly' button allows you to quickly set all the available paths (and control) to the same percentage.
+
+> Note: If Canvas reeligibility is enabled, users who enter the canvas and go down a randomly chosen path will go down the same path thereafter if they become re-eligible and re-enter the Canvas. This maintains the validity of the experiment and associated analytics.
```
Comment (EVERROCKET) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step.md:31]: Is it reeligibility or re-eligibility?

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**